staging/src/k8s.io/client-go/plugin/pkg/client/auth/exec/exec.go

```diff
@@ -191,40 +231,58 @@ func (r *roundTripper) RoundTrip(req *http.Request) (*http.Response, error) {
 	return res, nil
 }
 
-func (a *Authenticator) tokenExpired() bool {
+func (a *Authenticator) credsExpired() bool {
 	if a.exp.IsZero() {
 		return false
 	}
 	return a.now().After(a.exp)
 }
 
 func (a *Authenticator) token() (string, error) {
+	creds, err := a.freshCreds()
```

Comment:
the name freshCreds() makes me think it is skipping the cache...

Reply:
Renamed a few methods to make it clearer